eng/docker/fedora.38/Dockerfile

```diff
@@ -4,14 +4,14 @@
 #
 
 # Dockerfile that creates a container suitable to build dotnet-cli
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-36-20220716171953-531d246
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-38-20231107041221-75bbd6b
```

Comment:
This should use `mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-38` so it always targets the latest version of the image.